spring-cloud-kubernetes-test-support/pom.xml

```diff
@@ -46,7 +46,7 @@
 
 		<dependency>
 			<groupId>org.testcontainers</groupId>
-			<artifactId>k3s</artifactId>
+			<artifactId>testcontainers-k3s</artifactId>
```

Comment:
k3s testcontainers maven coordinates have changed, let's use the most recent ones